devel/darwinbuild-legacy/Portfile

```diff
@@ -1,53 +1,51 @@
 PortSystem      1.0
+PortGroup       github 1.0
 
 name            darwinbuild-legacy
 version         0.8.0
-revision	655
```

Comment:
You are effectively lowering the version (except that `github.setup` probably overrides the version field). Either version or revision should be higher that it was.